holoviews/plotting/bokeh/element.py

```diff
@@ -997,10 +1000,13 @@ def _setup_autorange(self):
             upper = None if (upper is None) or np.isnan(upper) else upper
 
         # Clean this up in bokeh 3.0 using View.find_one API
-        self.state.js_on_event('rangesupdate', CustomJS(code=f"""
-          const ref = cb_obj.origin.id
+        callback = CustomJS(code=f"""
+        const cb = function() {{
+          const ref = plot.id
+
           const find = (view) => {{
-            for (const sv of view.child_views) {{
+            let iterable = view.child_views === undefined ? []  : view.child_views
```

Comment:
One thing I missed is that it would be nice if this could traverse through Tabs which I think don't have child_views directly (although they might).

Reply:
Seems like that should work.

Reply:
Tried the following test with tabs and autoranging is still working!

<img width="679" alt="image" src="https://user-images.githubusercontent.com/890576/217001852-79fbf19c-a541-4d29-8a73-1669e9aa3efb.png">